devices/bmp3xx.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
+
+from absl import logging
+from . import Sensor
+
+import board
+import adafruit_bmp3xx
+
+
+class Bmp3xx(Sensor):
+  def __init__(self, remotestorage, localstorage, timesource, **kwargs):
+    super().__init__(remotestorage, localstorage, timesource)
+    self.sensor = adafruit_bmp3xx.BMP3XX_I2C(board.I2C())
+
+  def publish(self):
+    logging.info('Publishing Bmp3xx to remote')
```

Comment:
Should we log any of the data with this?

Reply:
I didn't because either way it was getting saved to a local database or pushed to remote.  Probably this is a low-value log message, though, and might be worth removing these to prevent log spewing.

Reply:
https://github.com/simpleairquality/SimpleAQ-Device/issues/39